tools/check-license-header.py

```diff
@@ -45,7 +45,7 @@
 
 comment_leading_chars = ('#', '::', '/*', '*', ' ')
 exclude_dirs = ('.git', 'lib', 'node_modules', 'release', '.idea', 'license')
-exclude_files = ('.md', '.DS_Store', 'NOTICE', 'LICENSE', '.gitignore', 'package.json', 'package-lock.json', '.prettierrc')
+exclude_files = ('.md', '.DS_Store', 'NOTICE', 'LICENSE', '.gitignore', 'package.json', 'package-lock.json', '.prettierrc', 'index.d.ts')
```

Comment:
This file should not be added as exclusive.

Reply:
i dont no how to solve the problem about "CI" check

Reply:
thank you ,I've solved it.